docs/user/monitoring/kibana-alerts.asciidoc

```diff
@@ -0,0 +1,37 @@
+[role="xpack"]
+[[kibana-alerts]]
+= {kib} Alerts
+
+The {stack} {monitor-features} provide
+<<alerting-getting-started,{kib} alerts>> out-of-the box to notify you of
+potential issues in the {stack}. These alerts are preconfigured based on the
+best practices recommended by Elastic. However, you can tailor them to meet your 
+specific needs.
+
+When you open the *{stack-monitor-app}* app, the preconfigured {kib} alerts are
+created automatically. If you collect monitoring data from multiple clusters,
+these alerts can search, detect, and alert on various conditions across the
+clusters. The alerts are visible alongside your existing {watcher} cluster
```

Comment:
Suggest changing the second use of alert in "these alerts can search, detect, and alert on various conditions"

Add a link to [Watcher](https://www.elastic.co/guide/en/kibana/current/watcher-ui.html)?

Reply:
@chrisronline Would it be technically accurate to change it to "..search, detect, and trigger actions on various conditions.."?

Reply:
Yea that's accurate, but sounds strange to stay "trigger actions". What if we just said "notify"?